src/app/components/system-maintenance/announcement/announcement-edit.jsx

```diff
@@ -1,44 +1,28 @@
 import React from 'react';
 import {
+  Box,
   FormControlLabel,
   FormHelperText,
   Switch,
   makeStyles,
 } from '@material-ui/core';
+
 import { func } from 'prop-types';
 import { useFormik } from 'formik';
 import * as yup from 'yup';
-
 import { ChplActionBar } from 'components/action-bar';
 import { ChplTextField } from 'components/util';
 import { jsJoda } from 'services/date-util';
 import { announcement as announcementPropType } from 'shared/prop-types';
 
-const useStyles = makeStyles({
+const useStyles = makeStyles(() => ({
```

Comment:
We've moved away from the `makeStyles` version that is a function
```suggestion
const useStyles = makeStyles({
```